integration_test/list_watch_test.go

```diff
@@ -1,29 +1 @@
 package integration_test
-
-import (
-	"context"
-	"time"
-
-	"github.com/telepresenceio/telepresence/v2/integration_test/itest"
-)
-
-func (s *connectedSuite) Test_ListWatch() {
```

Comment:
Why is this test removed?

Reply:
I'm trying to debug :( sometime the tests pass, all green, sometime they don't. Don't pay attention to the last commit.